Databricks/Alluxio-2.9.0-without-CVE/build-from-src-and-fix-CVE.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+set -e
+
+# clone, apply patches and compile
+cd /tmp/Alluxio-2.9.0-without-CVE
+git config --global user.email "you@example.com"
+git config --global user.name "Your Name"
+git clone https://github.com/Alluxio/alluxio.git
+cd alluxio/
+git checkout v2.9.0 -b v2.9.0-fix-cve
+git am < /tmp/Alluxio-2.9.0-without-CVE/patches/0001-Fix-CVE-issues.patch
+git am < /tmp/Alluxio-2.9.0-without-CVE/patches/0002-Fix-CVE-issues.patch
```

Comment:
can we just have this apply any patches in that directory and do it based on number order?

Reply:
Done